heed/src/databases/database.rs

```diff
@@ -2646,6 +2646,74 @@ impl<KC, DC, C, CDUP> Database<KC, DC, C, CDUP> {
         }
     }
 
+    /// Removes this database entirely.
+    /// The database is removed once the txn is committed.
```

Comment:
That's better for the doc. The first sentence will be used when a short text is required.

```suggestion
    /// Removes this database entirely.
    ///
    /// The database is removed once the txn is committed.
```

Reply:
Done!